index.html

```diff
@@ -45,7 +45,185 @@ <h1 class="banner_content">I’m Munish. <br>Glad to see you!</h1>
+                <div class="work_card_wrapper">
+                    <img src="images/work-3.png" alt="Work 3" class="work_img">
+                    <div class="work_card_content_wrapper">
+                        <h3 class="work_title">Tonic</h3>
+                        <ul class="work_tags">
+                            <li class="work_tags main_tag">Canopy</li>
+                            <li class="work_tags">Back End Dev</li>
+                            <li class="work_tags">2015</li>
+                        </ul>
+                        <p class="work_description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="work_tech_stack">
+                            <li class="work_tech_stack">HTML</li>
+                            <li class="work_tech_stack">CSS</li>
+                            <li class="work_tech_stack">JavaScript</li>
+                        </ul>
+                        <a href="index.html" class="work_link">
+                            <button class="work_button">See project</button>
+                        </a>
+                    </div>
+                </div>
+            </div> 
+            <div class="work_card">
+                <div class="work_card_wrapper">
+                    <img src="images/work-4.png" alt="Work 1" class="work_img">
+                    <div class="work_card_content_wrapper">
+                        <h3 class="work_title">Multi-Post Stories</h3>
+                        <ul class="work_tags">
+                            <li class="work_tags main_tag">Canopy</li>
+                            <li class="work_tags">Back End Dev</li>
+                            <li class="work_tags">2015</li>
+                        </ul>
+                        <p class="work_description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="work_tech_stack">
+                            <li class="work_tech_stack">HTML</li>
+                            <li class="work_tech_stack">CSS</li>
+                            <li class="work_tech_stack">JavaScript</li>
+                        </ul>
+                        <a href="index.html" class="work_link">
+                            <button class="work_button">See project</button>
+                        </a>
+                    </div>
+                </div>
+            </div>  
+        </section>
+
+        <!-- Works Section Ends Here -->
+
+        <!-- About Section Starts Here -->
+
+        <section class="about">
+            <div class="about_content_wrapper">
+                <h3 class="about_content">About Myself</h3>
+                <p class="about_content">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
```

Comment:
- [ ] Please update the about border radius to match the figma design. 

**Your project**
![image](https://user-images.githubusercontent.com/74411135/235277233-af4e2247-4ce4-4fb4-b354-ed937d545cef.png)

**figma design**
![image](https://user-images.githubusercontent.com/74411135/235277253-a758c23c-3fc0-4bcd-8561-3e6325ce28e0.png)

That will make your project look way better as the Figma design is done by a professional designer 😮 !